stob/module.c

```diff
@@ -23,28 +23,28 @@
 #include "stob/module.h"
 #include "module/instance.h"
 
-static int level_stob_enter(struct m0_module *module);
-static void level_stob_leave(struct m0_module *module);
+M0_UNUSED static int level_stob_enter(struct m0_module *module);
+M0_UNUSED static void level_stob_leave(struct m0_module *module);
 
-static const struct m0_modlev levels_stob[] = {
+M0_UNUSED static const struct m0_modlev levels_stob[] = {
 	[M0_LEVEL_STOB] = {
 		.ml_name = "stob is initialised",
 		.ml_enter = level_stob_enter,
 		.ml_leave = level_stob_leave,
 	}
 };
 
-static int level_stob_enter(struct m0_module *module)
+ M0_UNUSED static int level_stob_enter(struct m0_module *module)
```

Comment:
Remove the leading space.

Please note, there is a space at the beginning of this line.
I think this is caused by copy & paste.